jmx-scraper/src/main/resources/activemq.yaml

```
@@ -0,0 +1,69 @@
+---
+rules:
+  - beans:
+      - org.apache.activemq:type=Broker,brokerName=*,destinationType=Queue,destinationName=*
+      - org.apache.activemq:type=Broker,brokerName=*,destinationType=Topic,destinationName=*
+    metricAttribute:
+      destination: param(destinationName)
+      broker: param(brokerName)
+    prefix: activemq.
+    mapping:
+      ProducerCount:
+        metric: producer.count
+        unit: "producers"
+        type: updowncounter
+        desc: The number of producers currently attached to the broker.
+      ConsumerCount:
+        metric: consumer.count
+        unit: "consumers"
+        type: updowncounter
+        desc: The number of consumers currently reading from the broker.
+      MemoryPercentUsage:
+        metric: memory.usage
+        unit: "%"
+        type: gauge
+        desc: The percentage of configured memory used.
+      QueueSize:
+        metric: message.current
+        unit: "messages"
+        type: updowncounter
+        desc: The current number of messages waiting to be consumed.
+      ExpiredCount:
+        metric: message.expired
+        unit: "messages"
+        type: counter
+        desc: The total number of messages not delivered because they expired.
+      EnqueueCount:
+        metric: message.enqueued
+        unit: "messages"
+        type: counter
+        desc: The total number of messages received by the broker.
+      DequeueCount:
+        metric: message.dequeued
+        unit: "messages"
+        type: counter
+        desc: The total number of messages delivered to consumers.
+      AverageEnqueueTime:
+        metric: message.wait_time.avg
+        unit: ms
+        type: gauge
+        desc: The average time a message was held on a destination.
+
+  - bean: org.apache.activemq:type=Broker,brokerName=*
+    metricAttribute:
+      broker: param(brokerName)
```

This attribute was present in original activemq.yaml. Because of this some minor changes has to be made in assertions to check this metric attribute

---

Maybe add an explicit comment like in the `tomcat.yaml` to explain the difference, adding one extra metric attribute should not have much impact and thus it should remain compatible.

```suggestion
      # minor divergence from activemq.groovy to capture broker name, making it closer to
      # the definition in JMX Insights
      broker: param(brokerName)
```